force-app/main/default/lwc/nksFagsystemer/nksFagsystemer.js

```diff
@@ -19,105 +21,156 @@ export default class NksFagsystemer extends LightningElement {
     @api title;
     @api relatedField;
     @api objectApiName;
-    @track showLinks;
-    @track inFagsone = false;
     @api filterList;
-    @api personId;
+    @api personId; // deprecated
 
+    inFagsone = false;
+    isSandbox = false;
+    showSpinner = false;
+    wiredObject;
+    actorId;
+    navIdent;
     hiddenLinks = ['Aktivitetsplan', 'Speil'];
+    _personIdent;
 
-    possibleLinks = [
-        { name: 'Modia', field: 'NKS_ModiaURL__c' },
-        { name: 'Gosys', field: 'NKS_GoSysURL__c' },
-        { name: 'Aktivitetsplan', field: 'NKS_AktivitetsplanURL__c' },
-        { name: 'AA-reg', field: null, eventFunc: this.handleAAClickOrKey, title: 'AA-register' },
-        { name: 'DinPensjon', field: 'NKS_PSELVPensjonURL__c' },
-        { name: 'DinUfore', field: 'NKS_PSELVUfoeretrygdURL__c' },
-        { name: 'Pesys', field: 'NKS_PESYSURL__c' },
-        { name: 'Speil', field: 'NKS_SpeilURL__c' },
-        { name: 'Foreldrepenger', field: 'NKS_ForeldrepengerURL__c' },
-        { name: 'K9', field: 'NKS_K9URL__c' },
-        { name: 'Barnetrygd', field: 'NKS_BarnetrygdURL__c' },
-        { name: 'Enslig', field: 'NKS_EnsligForsorgerURL__c' },
-        { name: 'Kontantstøtte', literalLink: 'https://kontantstotte.intern.nav.no/' }
-    ];
-    // {
-    //     name: 'Sosial',
-    //     field: null,
-    //     eventFunc: this.handleSosialModiaClickOrKey,
-    //     title: 'Modia Sosialhjelp',
-    //     show: NKS_SosialTilgang
-    // },
-    // { name: 'SYFO', field: null, eventFunc: this.handleSYFOClickOrKey, title: 'SYFO' },
-
-    connectedCallback() {
-        checkFagsoneIpRange().then((res) => {
-            this.inFagsone = res.isInFagsone;
-            if (this.inFagsone === false) {
-                console.log('Ip is: ' + res.ip);
-            }
-        });
+    @api
+    get personIdent() {
+        return this._personIdent;
     }
 
-    renderedCallback() {
-        const listOfFilter =
-            typeof this.filterList === 'string' ? this.filterList.replaceAll(' ', '').split(',') : this.filterList;
-        this.fields = this.possibleLinks
-            .map((link, index) => ({
-                ...link,
-                id: index,
-                custom: link.field == null,
-                show: !('show' in link) || (link.show ?? false)
-            }))
-            .filter(filterFunc(this.hiddenLinks, listOfFilter));
+    set personIdent(value) {
+        this._personIdent = value;
     }
 
-    get size() {
+    get layoutItemSize() {
         return 6;
     }
 
-    @wire(getPersonId, {
+    get showContent() {
+        return this.wiredObject != null;
+    }
+
+    get showRefreshButton() {
+        return !(!this.objectApiName || !this.recordId || !this.relatedField);
+    }
```

Comment:
Dobbelt negativt er ganske dumt
```suggestion
    get showRefreshButton() {
        return this.objectApiName && this.recordId && this.relatedField;
    }
```

Reply:
Dette er faktisk din kode som jeg ikke så så nøye på skal jeg være helt ærlig 😄 Men enig.